package.json

```diff
@@ -31,32 +31,32 @@
   },
   "homepage": "https://github.com/Half-Shot/matrix-appservice-discord#readme",
   "dependencies": {
-    "@types/bluebird": "^3.0.37",
-    "@types/node": "^7.0.5",
+    "@types/bluebird": "^3.5.20",
+    "@types/node": "^7.0.55",
     "@types/sqlite3": "^2.2.32",
-    "bluebird": "^3.4.7",
+    "bluebird": "^3.5.1",
     "command-line-args": "^4.0.1",
-    "command-line-usage": "^4.0.0",
-    "discord.js": "^11.1.0",
+    "command-line-usage": "^4.1.0",
+    "discord.js": "^11.3.0",
```

Comment:
specifically, discord.js being upgraded fixes the issue described in https://github.com/Half-Shot/matrix-appservice-discord/pull/85

Reply:
Does it need to be bumped past 11.1.0, same for command-line-usage?

Reply:
just discord.js being upgraded from 11.1.0 to something newer fixed my issue. This PR is more the result of me being lazy and just running `npm upgrade`.